packages/botonic-react/src/webchat/devices/scrollbar-controller.js

```diff
@@ -0,0 +1,103 @@
+import { DEVICES, isMobileDevice } from '.'
+import { getWebchatElement, getScrollableArea } from '../../utils'
+
+const debounced = (delay, fn) => {
+  let timerId
+  return function (...args) {
+    if (timerId) {
+      clearTimeout(timerId)
+    }
+    timerId = setTimeout(() => {
+      fn(...args)
+      timerId = null
+    }, delay)
+  }
+}
```

Comment:
Do you include `lodash` or `underscore` on botonic? They should include this feature, but I might be wrong. If not, that's okay

Reply:
We don't use lodash neither underscore right now in Botonic, but maybe we should consider it! What do you think about?